core/src/test/java/org/apache/calcite/test/JdbcFrontJdbcBackTest.java

```diff
@@ -73,6 +79,75 @@ class JdbcFrontJdbcBackTest {
         });
   }
 
+
+  /**
+   * Sample subclass used in {@link JdbcFrontJdbcBackTest#testTablesExtraColumn()}.
+   */
+  public static class MetaExtraTable extends CalciteMetaTable {
```

Comment:
Does this class need to be public? Does its field need to be public?

If it can be private, that's good. If it needs to be public (because, say, reflection) then you should say that it needs to be public in the javadoc.